### DISCUSSION TITLE
Navigating the calendar by year, not just by month.

### DISCUSSION THREAD

**Discussion by tomleary** [liamcain/obsidian-calendar-plugin#200] [Ideas]
I have a bunch of journal entries going back years and I don't mind entering them in one at a time, but it's obnoxious to have to click the previous month button 48 times to get to 2017. Is it possible to navigate the calendar by year? If not, can this be added in a future update?

**Comment by roweger** [comment]
Came here for this exact reason. Please please implement this

**Comment by roweger** [comment]
Discovered this was requested as issue #61 last year.
